bin/coadd_zint

```diff
@@ -37,12 +39,15 @@ h.close()
 da = {}
 we = {}
 
-h = fitsio.FITS(args.data[0].replace('cf','dmat'))
-dm = h[1]['DM'][:]*0
-h.close()
+if not args.no_dmat:
+    h = fitsio.FITS(args.data[0].replace('cf','dmat'))
+    dm = h[1]['DM'][:]*0
+    h.close()
+else:
+    dm = sp.zeros((nb.shape[0],nb.shape[0]))
```

Comment:
dm = sp.eye(nb.shape[0])

Reply:
Line 45 sets dm to an array of zeros if you do have a dmat, so surely line 47 should also set it to an array of zeros?

Reply:
But if you remove lines 76 and 77, it would work perfectly.